lib/wallet/bitcoin_tx_detail.dart

```diff
@@ -39,32 +39,41 @@ class BitcoinTxDetail extends StatelessWidget {
       body: SafeArea(
         child: Container(
             padding: const EdgeInsets.all(20.0),
-            child: TtoTable([
-              TtoRow(
-                  label: "Transaction Id",
-                  value: transaction.txid.substring(0, 7),
-                  type: ValueType.link,
-                  meta: explorerUrl + "/" + transaction.txid),
-              TtoRow(
-                  label: "Timestamp",
-                  value: DateFormat('dd.MM.yy-kk:mm')
-                      .format(DateTime.fromMillisecondsSinceEpoch(transaction.timestamp * 1000)),
-                  type: ValueType.date),
-              TtoRow(
-                  label: "Fees", value: formatter.format(transaction.fee), type: ValueType.satoshi),
-              TtoRow(
-                  label: "Confirmed",
-                  value: transaction.isConfirmed.toString(),
-                  type: ValueType.text),
-              TtoRow(
-                  label: "Sent",
-                  value: formatter.format(transaction.sent),
-                  type: ValueType.satoshi),
-              TtoRow(
-                  label: "Received",
-                  value: formatter.format(transaction.received),
-                  type: ValueType.satoshi)
-            ])),
+            child: Column(
+              children: [
+                TtoTable([
+                  TtoRow(
+                      label: "Transaction Id",
+                      value: transaction.txid.substring(0, 7),
+                      type: ValueType.link,
+                      meta: explorerUrl + "/" + transaction.txid),
+                  TtoRow(
+                      label: "Timestamp",
+                      value: DateFormat('dd.MM.yy-kk:mm').format(
+                          DateTime.fromMillisecondsSinceEpoch(transaction.timestamp * 1000)),
+                      type: ValueType.date),
+                  TtoRow(
+                      label: "Fees",
+                      value: formatter.format(transaction.fee),
+                      type: ValueType.satoshi),
+                  TtoRow(
+                      label: "Confirmed",
+                      value: transaction.isConfirmed.toString(),
+                      type: ValueType.text),
+                  TtoRow(
+                      label: "Sent",
+                      value: formatter.format(transaction.sent),
+                      type: ValueType.satoshi),
+                  TtoRow(
+                      label: "Received",
+                      value: formatter.format(transaction.received),
+                      type: ValueType.satoshi)
```

Comment:
We could combine this in one row and only show either of them depending on this being a send or receive transaction.